<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [iOSSuite]">
  <listeners>
    <listener class-name="ios.common.functions.RetryListener"/>
  </listeners>
  <test name="iOSTest(failed)">
    <classes>
      <class name="ios.scripts.Test_04_UI_Catalog_AlertViews">
        <methods>
          <include name="after_Suite"/>
          <include name="test_04_UI_Catalog_AlertViews" invocation-numbers="0 0"/>
          <include name="before_Suite"/>
          <include name="before_Method"/>
          <include name="after_Method"/>
        </methods>
      </class> <!-- ios.scripts.Test_04_UI_Catalog_AlertViews -->
    </classes>
  </test> <!-- iOSTest(failed) -->
</suite> <!-- Failed suite [iOSSuite] -->
